src/prevector.h

```diff
@@ -220,7 +220,10 @@ class prevector {
         }
     }
 
-    prevector() : _size(0) {}
+    prevector() : _size(0) {
+        // Explicitly initialize indirect ptr to avoid "used uninitialized" warning
+        _union.indirect = nullptr;
```

Comment:
I was vaguely concerned that this assignment violates type punning through a union, but (1) some sources say that is permitted anyway (2) the other type is char (for which exceptions to type punning exist) and (3) we'll write to the union again before ever using data in it.